src/modules/site-v2/base/static/js/plot.js

```diff
@@ -269,6 +285,10 @@ function render_scatterplot_histograms(container_selector, data, config={}) {
  */
 function render_histogram(container_selector, data, config={}) {
 
+  if (data.length == 0) {
```

Comment:
prefer triple strict equals `===` to ensure that both type and value are compared. This avoids type casting side effects.